tests/src/Functional/RequestTest.php

```diff
@@ -0,0 +1,982 @@
+<?php
+
+namespace Drupal\Tests\schemataFunctional;
+
+use Behat\Mink\Driver\BrowserKitDriver;
+use Drupal\Core\Url;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\node\Entity\NodeType;
+use Drupal\taxonomy\Entity\Vocabulary;
+use Drupal\Tests\BrowserTestBase;
+use GuzzleHttp\RequestOptions;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Tests requests schemata routes.
+ *
+ * @group Schemata
+ */
+class RequestTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['text', 'node', 'taxonomy', 'serialization'];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    if (!NodeType::load('camelids')) {
+      // Create a "Camelids" node type.
+      NodeType::create([
+        'name' => 'Camelids',
+        'type' => 'camelids',
+      ])->save();
+    }
+
+    // Create a "Camelids" vocabulary.
+    $vocabulary = Vocabulary::create([
+      'name' => 'Camelids',
+      'vid' => 'camelids',
+    ]);
+    $vocabulary->save();
+
+    $entity_types = ['node', 'taxonomy_term'];
+    foreach ($entity_types as $entity_type) {
+      // Add access-protected field.
+      FieldStorageConfig::create([
+        'entity_type' => $entity_type,
+        'field_name' => 'field_test_' . $entity_type,
+        'type' => 'text',
+      ])
+        ->setCardinality(1)
+        ->save();
+      FieldConfig::create([
+        'entity_type' => $entity_type,
+        'field_name' => 'field_test_' . $entity_type,
+        'bundle' => 'camelids',
+      ])
+        ->setLabel('Test field')
+        ->setTranslatable(FALSE)
+        ->save();
+    }
+    // @todo Schemata Routes aren't rebuilt after new content type.
+    $this->container->get('module_installer')->install(['schemata', 'schemata_json_schema'], TRUE);
```

Comment:
Why install them now instead of on the `$modules` property? Would it make sense to _just_ rebuild routes here instead?